sphinx_scality/static/css/current.css_t

```diff
@@ -269,4 +275,19 @@ div.footer div.rightwrapper img.social {
 .term {
     color: #F37533;
     font-weight: bold;
+}
+
+/* search box */
+
+input[type=text], select {
```

Comment:
We need a more precise selector here, and I'm quite surprised by the `select` here, is it needed?